src/pages/Questions/Question.jsx

```diff
@@ -0,0 +1,148 @@
+import "../../css/main.css";
+import "./Question.css";
+import { Link, useParams } from "react-router-dom";
+import { useQuiz } from "../../context";
+import { Loader } from "../../components";
+import { useState, useEffect } from "react";
+import { useScrollToTop, useDocumentTitle } from "../../utils";
+
+export const Question = () => {
+  useScrollToTop();
+  useDocumentTitle("Cric Quiz | Question");
+  const { selectedQuiz, answers, setAnswers } = useQuiz();
+  const { selectedQuizData } = selectedQuiz;
+  let { questionNumber } = useParams();
+  const [questionData, setQuestionData] = useState({});
+  const [selectedAnswer, setSelectedAnswer] = useState("");
+
+  const quizId = selectedQuizData._id;
+  const getQuestionData = () => {
+    const requiredQuestion = selectedQuizData.questions[questionNumber];
+    setQuestionData(requiredQuestion);
+    setSelectedAnswer("");
+  };
+
+  useEffect(() => {
+    getQuestionData();
+  }, [questionNumber, questionData]);
+
+  const answersUpdateHandler = () => {
+    setAnswers([...answers, selectedAnswer]);
+  };
+
+  console.log(answers);
+
+  return (
+    <main className="flex flex-grow-1 flex-justify-center flex-align-center">
+      {Object.keys(questionData).length === 0 && <Loader />}
+      {Object.keys(questionData).length !== 0 && (
```

Comment:
Similarly, here also ternary operator can be used to check `Object.keys(questionData).length` only once.